service/implementation/auto_request_api/logic_auto_request.py

```diff
@@ -425,4 +440,13 @@ def auto_request_system(api: Dict[str, str]) -> None:
     except requests.exceptions.RequestException as e:
         print(f"Помилка при запиті до {api['name']}: {e}")
     except Exception as e:
-        print(f"Загальна помилка при збереженні даних для {api['name']}: {e}")
\ No newline at end of file
+        print(f"Загальна помилка при збереженні даних для {api['name']}: {e}")
+
+
+def keep_db_alive():
+    session = scoped_session(SessionLocal)
+    try:
+        session.execute(text("SELECT 1"))
+        session.commit()
+    except Exception as e:
+        print(f"Keep-alive failed: {e}")
```

Comment:
some text/exceptions is/are in Ukrainian and others in English - that should be fixed. Use one style and language throughout all your project files